src/store/accounts/nonce.ts

```diff
@@ -21,11 +23,22 @@ export const latestNonceState = atom(get => {
   return get(localNoncesState([network.url, address || '']));
 });
 
+export const lastApiNonce = atom(async get => {
```

Comment:
```suggestion
export const lastApiNonce = atom(get => {
```

No need for `async` because it returns a promise, only need `async` if there is an `await` in the body

Reply:
There were an await in my previous code. I missed removing this.